### ISSUE TITLE
Keras model import: handle training configuration properly

### ISSUE THREAD

**Issue by maxpumperla** [deeplearning4j/deeplearning4j#3936]
#### Issue Description

Currently we only support parts of the training configuration correctly. This is insofar OK as we're mainly doing this whole exercise to import pre-trained models. That would include:
- loss weights
- sample weights
- optimizer configs
